### PR TITLE
usb:update usb command for rhel9 guest

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:1985938 
Signed-off-by: MiriamDeng <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
Test results,
2021-07-26 09:29:05 | 1 | Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.usb_device_check.multi_hubs_in_series.5_layer.nec-xhci |   | PASS | 112.14 |   |  
2021-07-26 09:30:57 | 2 | Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.usb_device_check.multi_hubs_in_series.5_layer.qemu-xhci |   | PASS | 113.99 |   |  
2021-07-26 09:32:51 | 3 | Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.usb_device_check.multi_hubs_in_parallel.nec-xhci |   | PASS | 117.39 |   |  
2021-07-26 09:34:49 | 4 | Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.usb_device_check.multi_hubs_in_parallel.qemu-xhci |   | PASS | 116.47

**Comment by MiriamDeng** [comment]
@ybduan @PaulYuuu ,can you help to review this patch ? Thanks a lot

File: qemu/tests/cfg/usb_device_check.cfg [hunk 1]
```diff
@@ -9,7 +9,7 @@
     login_timeout = 180
     type = usb_device_check
     chk_usb_info_cmd = "lsusb"
-    RHEL.7, RHEL.8:
+    RHEL.7, RHEL.8, RHEL.9:
```
Comment (yanan-fu) [inline qemu/tests/cfg/usb_device_check.cfg:12]: Suggest use `chk_usb_info_cmd = "lsusb -v"` as the default setting, compatible with the new version.

Comment (MiriamDeng) [reply qemu/tests/cfg/usb_device_check.cfg:12]: It's fine with me, but I want to confirm it with @ybduan , what do you think ?

**Comment by ybduan** [comment]
This solution isn't acceptable and conflicted with commit 17164f3f (PR #2793 ).
The root cause is that the outputs of 'lsusb' and 'lsusb -v' are changed in latest RHEL for usb-mouse/usb-kbd/usb-tablet devices, but not usb-hub.
So PR #2793 is not perfect (missing usb-hub consideration). I think we should adjust function 'verify_usb_device_in_guest' in 'usb_common.py' according to the latest changes.

**Comment by MiriamDeng** [comment]
> This solution isn't acceptable and conflicted with commit [17164f3](https://github.com/autotest/tp-qemu/commit/17164f3f6a1a7fe9c0feb83d6fd8019b91e08dae) (PR #2793 ).
> The root cause is that the outputs of 'lsusb' and 'lsusb -v' are changed in latest RHEL for usb-mouse/usb-kbd/usb-tablet devices, but not usb-hub.
> So PR #2793 is not perfect (missing usb-hub consideration). I think we should adjust function 'verify_usb_device_in_guest' in 'usb_common.py' according to the latest changes.

Ok, thanks for you mentioned it.

**Pull request closed without merging**